### PR TITLE
Allow sentry logging to work correctly from the websocket server

### PR DESCRIPTION
**Pull request by finitud**

### CODE REVIEW

**Comment by nickstenning** [comment]
This looks good, and will stop the websocket server crashing, but you also need to update the `get_reader` call in `process_nsq_topic` to pass in a Sentry client of its own construction!

File: h/queue.py [hunk 1]
```diff
@@ -92,6 +90,16 @@ def resolve_topic(topic, namespace=None, settings=None):
     return topic
 
 
+def _get_queue_reader(request, topic, channel):
+    get_reader(request.registry.settings,
```
Comment (nickstenning) [inline h/queue.py]: Oops! You're missing a `return` here.

Comment (finitud) [inline h/queue.py]: Whoops! Keep forgetting Python functions don't just return the last value!

**Comment by nickstenning** [comment]
LGTM.

**Pull request merged**